assets/brunch-config.js

```diff
@@ -0,0 +1,67 @@
+exports.config = {
```

Comment:
do we _need_ brunch in this project?
(_are we doing any JS or CSS compilation_?)
Or is it just for page reloading?

Reply:
We don't necessarily need brunch right now, but we are using it to build our static files by default. We could take some time to remove it if necessary

Reply:
Now that we're using Elm, brunch is a bit more of a necessity to compile it with our javascript. We could still replace it with something else, but unless there's a clear advantage, I'm happy to stick with it

Reply:
Happy to use Brunch for now. 👍 
Phoenix 1.4 uses Webpack: https://github.com/phoenixframework/phoenix/blob/0bdb5b001915e437c2d2cf1b77a1c532ce45cde2/CHANGELOG.md#phxnew-installer
🙄